packages/services/api/src/modules/schema/providers/schema-manager.ts

```diff
@@ -1007,12 +1007,11 @@ export class SchemaManager {
     });
 
     await this.session.assertPerformAction({
-      action: 'schema:loadFromRegistry',
+      action: 'project:describe',
```

Comment:
`schema:loadFromRegistry` is too specific. Instead ,it should be `project:describe` access that must be granted to the access token. Since we do not expose the permission to the outside world, this change is safe to perform.

Reply:
Agreed. I dont see a case where someone should be able to access the project but not the schema.